### PR TITLE
Edit descriptions on the local wiki for English.

### PR DESCRIPTION
**Pull request by dbrant**

### CODE REVIEW

File: app/src/main/java/org/wikipedia/descriptions/DescriptionEditFragment.java [hunk 1]
```diff
@@ -415,4 +478,36 @@ public void onVoiceInputClick() {
             }
         }
     }
+
+    private String updateDescriptionInArticle(String articleText, String newDescription) {
```
Comment (cooltey) [inline app/src/main/java/org/wikipedia/descriptions/DescriptionEditFragment.java:482]: Would it be better if add `@NonNull` to the parameters?

File: app/src/main/java/org/wikipedia/descriptions/DescriptionEditUtil.java [hunk 2]
```diff
@@ -15,6 +15,11 @@
 
     public static boolean isEditAllowed(@NonNull Page page) {
         PageProperties props = page.getPageProperties();
+        if (page.getTitle().getWikiSite().languageCode().equals("en")) {
```
Comment (cooltey) [inline app/src/main/java/org/wikipedia/descriptions/DescriptionEditUtil.java:18]: Maybe it is redundant but is it a good idea to put `&& props.canEdit()` into the `if` statement?

File: app/src/main/java/org/wikipedia/descriptions/DescriptionEditFragment.java (hunk 1 above)
Comment (dbrant) [reply app/src/main/java/org/wikipedia/descriptions/DescriptionEditFragment.java]: yep

File: app/src/main/java/org/wikipedia/descriptions/DescriptionEditUtil.java (hunk 2 above)
Comment (dbrant) [reply app/src/main/java/org/wikipedia/descriptions/DescriptionEditUtil.java:18]: Technically this method tells if the *description* specifically should be editable. I'm not sure we should mix up whether the entire article is editable here. (it's already handled correctly elsewhere)

**Review by cooltey — APPROVED** [review]
Tested and looks good to me. @sharvaniharan Please review and merge if it looks good to you too.

**Pull request merged**